### DISCUSSION TITLE
AAI drive not connecting to 2022 BMW X5 G05

### DISCUSSION THREAD

**Discussion by Vlad777777** [BimmerGestalt/AAIdrive#781] [Q&A]
I used AAIdrive with my 2017 BMW X5 F15 and it worked great!
But after I upgraded my car to 2022 BMW X5 G05, AAI does not connect, it still has my previous car data in it.
I uninstalled the app and reinstalled, but my old car data is still in the app and it does not connect.
Apparently, uninstall did not wipe out my old car data.
Also, my old BMW did not have Android Auto, but the new one has it in drive 7.
Any suggestions how to fix this. How to uninstal AAI with data wipe?
Thanks,
Vladimir  <vshulyaker@gmail.com>

**Comment by hufman** [comment]
AAIdrive doesn't use the stored data when connecting to the car, so there's no harm in it showing the old data.
IDrive7 does not support AAIdrive's protocol. Certain cheaper 2022 BMWs still use IDrive6 and do work with AAIdrive, but yours is too fancy.

**Reply by Vlad777777** [reply]
Thanks, no luck for me...
The "built-in Android Auto does not give the functionality to integrate apps like calendar into iDrive7 display.
Also, myBMW app is very limited as well..
Do you have any plans to have an update that would work with later "fancy" BMW iDrive7?
(Apparently not fancy enough to show functionality available before...)

**Reply by hufman** [reply]
Unfortunately, there is no way for AAIdrive to connect to Idrive7, BMW turned off the Apps protocol in that generation.
